any_parser/utils.py

```diff
@@ -8,6 +8,7 @@
 class ModelType(Enum):
     BASE = "base"
     PRO = "pro"
+    PARSE_WITH_LAYOUT = "parse_with_layout"
```

Comment:
like above. let's have a separated parse_with_layout instead of nesting parse_with_layout in current logic.

Reply:
ack